src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -95,6 +97,13 @@ def _safe_int(num):
             )
 
 
+def _point_coordinate(value):
+    if not value or not isinstance(value, list) or len(value) != 2:
+        return SKIP
+    # cast to str because floats would get localized in the template
```

Comment:
shouldn't we just disable localization in the template then? https://docs.djangoproject.com/en/2.2/topics/i18n/formatting/#localize

Reply:
I forgot about the `unlocalize` filter.